crawlerr/config.py

```
@@ -1,104 +1,133 @@
 """
 Configuration management for Crawlerr using Pydantic Settings.
 """
-from typing import Optional, List
 from pydantic_settings import BaseSettings
-from pydantic import Field, validator
+from pydantic import Field, field_validator, ConfigDict
 import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)